xrpl/models/requests/simulate.py

```diff
@@ -0,0 +1,71 @@
+"""This request simulates a transaction without submitting it to the network."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, Optional, Type
+
+from typing_extensions import Self
+
+from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.utils import KW_ONLY_DATACLASS, require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class Simulate(Request):
+    """
+    The `simulate` method simulates a transaction without submitting it to the
+    network.
+    """
+
+    tx_blob: Optional[str] = None
```

Comment:
Could you add docstring descriptions for these variables?

Reply:
Fixed